### ISSUE TITLE
suggestion: comparing operators

### ISSUE THREAD

**Issue by doubledusk** [Aceship/AN-EN-Tags#48]
certain operators (Silence and Hellagur for example) have varying attack speed from their traits. i'd like to suggest a slider to change filters based on this, if possible. for example, to see what Hellagur's dps is when he's at different HP thresholds. similarly, Exusiai and Skadi will always have a +% of attack added to their damage due to traits, making their DPS higher than what their raw stats say. something on the menu to allow these things to be added / shown would be nice

**Comment by EdeEe13** [comment]
i agree with the suggestion that traits/talents should be added as they make a huge difference in statts.

also would love option to adjust in potential as they also have stattincrease or talent increase 

just looking at defenders cuora and bison as easy obtainable operators [in addtion with hosihiguma and nian] have traits that boost def/hp

in poor def without and with talents [not counting in the potentials]
cuora E2 70 : 765 > 856,8
bison E2 80: 781 >821(841 potential bonus of talent) 
Nian E2 90:  796 

as it can be seen atm it would look like that nian has more def than either bison or cuora but due to talents alone we can see that nian would lose in defence comparison [potential on lower rarity or event units will make even more difference as they are easier to get potential X to stattinrease]

i mean i would also love to see option to scale +%statts due to skills but think that might be just tomuch work
